### PR TITLE
chore: Add @StackScribe to CONTRIBUTORS.md

### PR DESCRIPTION
**Pull request by StackScribe**

### CODE REVIEW

File: helm-service/chart/values.yaml [hunk 1]
```diff
@@ -7,9 +7,9 @@ helmservice:
     enabled: true                            # Creates a Kubernetes Service for the helm-service
 
 distributor:
-  stageFilter: ""                            # Sets the stage this helm service belongs to
-  serviceFilter: ""                          # Sets the service this helm service belongs to
-  projectFilter: ""                          # Sets the project this helm service belongs to
+  stageFilter: ""                            # Stage to which this helm service belongs; default is all; comma-separated list to filter for set of stages
```
Comment (oleg-nenashev) [inline helm-service/chart/values.yaml:10]: Unrelated

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -78,3 +78,4 @@ For a realtime overview of current contributors to the Keptn project, we refer t
 * [Luis Marsano](https://github.com/lmmarsano)
 * [Oleg Nenashev](https://github.com/oleg-nenashev)
 * [Stuart Blackler](https://github.com/Im5tu)
+* [Meg McRoberts](https://github.com/Im5tu)
```
Comment (oleg-nenashev) [inline CONTRIBUTORS.md:81]: Wrong GitHub ID

**Review by oleg-nenashev — CHANGES_REQUESTED** [review]
The pull request needs to be fixed. I also recommend following the process in https://github.com/keptn/community/blob/main/COMMUNITY_MEMBERSHIP.md if you want to join the community as an official member

**Comment by mowies** [comment]
This PR needs a rebase...

**Pull request closed without merging**